### PR TITLE
changed navbar to green

### PR DESCRIPTION
**Pull request by tonklis**
Changed the navbar to green to help colorblind people

### CODE REVIEW

File: app/assets/stylesheets/components/_navbar.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .navbar-lewagon {
   justify-content: space-between;
-  background: white;
+  background: green;
```
Comment (tonklis) [inline app/assets/stylesheets/components/_navbar.scss]: why green?

Comment (tonklis) [inline app/assets/stylesheets/components/_navbar.scss]: green is my fav color!

**Comment by mshin19** [comment]
Test

File: app/assets/stylesheets/components/_navbar.scss (hunk 1 above)
Comment (mshin19) [reply app/assets/stylesheets/components/_navbar.scss]: test 2

**Comment by tonklis** [comment]
Merging after 04/04/2019 12:00pm meeting

ping @tonklis

**Pull request merged**